README.md

```diff
@@ -27,15 +28,13 @@ Below are the steps needed to start using the System.Web adapters with your ASP.
      </packageSources>
    </configuration>
    ```
-2. Install `Microsoft.AspNetCore.SystemWebAdapters`
-3. If you use `HttpContext.Session`, install `Microsoft.AspNetCore.SystemWebAdapters.SessionState`
-4. In your framework application:
-   - The package installation will add a new module to your `web.config`. This module handles any customizations that are required to help migrate to .NET Core. See [this](docs/framework.md) for details on what is available here.
-5. In your class libraries:
+2. Install `Microsoft.AspNetCore.SystemWebAdapters` to supporting libraries
    - Class libraries can target .NET Standard 2.0 if desired which will ensure you are using the shared surface area
    - If you find that there's still some missing APIs, you may cross-compile with .NET Framework to maintain that behavior and handle it in .NET core in some other way
    - There should be no manual changes to enable using supported surface area of the adapters. If a member is not found, it is not currently supported on ASP.NET Core
-6. For your ASP.NET Core application:
+3. Install `Microsoft.AspNetCore.SystemWebAdapters.Extensions` to your ASP.NET Core and Framework applications
```

Comment:
Needs updating

Reply:
I had, but apparently hadn't saved it before I pushed it up....